src/frontend/web_application/src/components/ImportContactForm/presenter.jsx

```diff
@@ -1,139 +1,188 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { v1 as uuidV1 } from 'uuid';
 import { FieldErrors } from '../form';
 import Button from '../Button';
 import Icon from '../Icon';
 
 import './style.scss';
 
-const VALID_EXT = ['vcf', 'vcard'];
+const VALID_EXT = ['vcf', 'vcard']; // Valid file extensions for input#file
+
+const File = ({ file, onClick }) => (
+  <div className="m-import-contact-form__file">
+    <span className="m-import-contact-form__file-name">{file.name}</span>
+    <span className="m-import-contact-form__file-size">{file.size / 1000} ko</span>
```

Comment:
Size must be localized: ( FR: 1 Ko, EN: 1 KB )

`{ __('import-contact.file.size', { size: file.size / 1000 }) }`

and in locales:

`"import-contact.file.size": "%(size)s KB"`

`"import-contact.file.size": "%(size)s Ko"`